templates/test/overview.html.ep

```diff
@@ -94,7 +94,11 @@
                 % for my $config (@$configs) {
                     % next unless $results->{$config}{$type};
                     <tr>
-                        <td class="name"><%= trim_text($config, 30) %></td>
+                        <!--<td class="name"><%= trim_text($config, $name_limit) %></td>-->
+                        % my $description = 'TODO fill with test suite description. But where to get it from without joining the test suite table onto the job results?';
```

Comment:
If you don't want a join slowing down loading the page, you can find the associated test suite via async AJAX call on popover. Note that the `MACHINE` column might be appended here, eg `btrfs@s390x-zVM-ctc`.